sources/packages/backend/workflow/test/jest-e2e.json

```diff
@@ -10,7 +10,7 @@
   "transform": {
     "^.+\\.(t|j)s$": "ts-jest"
   },
-  "testTimeout": 60000,
+  "testTimeout": 90000,
```

Comment:
was it taking more time? just to check

Reply:
Sometimes(observed only in local and **_not_** in GitHub actions) that assessment gateway takes little more than 60 seconds to complete. Hence increased the timeout to ensure the tests does not timeout waiting for workflow to complete.